pipelines/skylab/slideseq/SlideSeq.changelog.md

```diff
@@ -1,8 +1,9 @@
 # 2.1.5
-2024-01-11 (Date of Last Commit)
+2024-01-24 (Date of Last Commit)
 
-* Increased memory for MergeStarOutputs in StarAlign.wdl, RunEmptyDrops in RunEmptyDrops.wdl, OptimusH5ad in H5adUtils.wdl and GeneMetrics in Metrics.wdl
```

Comment:
Are these updates also included in this version? If so, let's not delete them, just add to them

Reply:
I think its in the previous version. I accidentally removed the points. Thanks for pointing that out!